src/Aquifer.API/Endpoints/Resources/Content/Versions/History/Endpoint.cs

```diff
@@ -0,0 +1,39 @@
+using Aquifer.Common.Extensions;
+using Aquifer.Data;
+using FastEndpoints;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Aquifer.API.Endpoints.Resources.Content.Versions.History;
+
+public class Endpoint(AquiferDbContext dbContext) : Endpoint<Request, List<Response>>
+{
+    public override void Configure()
+    {
+        Get("/resources/content/versions/history/{VersionId}");
```

Comment:
This History folder should go inside the Get folder, because the path already has an individual version on it, and now you're asking for something inside that. So `/resources/content/versions/{VersionId}/history`.

I'm not sure if history is quite right in this case though, since it's an assignment and status history, and status doesn't even live on the version, it lives on ResourceContent. Assignment feels like it could be a type of status for the item. I don't know if this is best, but I would lean into something like `status-history` rather than just history.

Reply:
I forgot the status history is on the version, even though the status itself is on the content.